src/robot/model/testsuite.py

```diff
@@ -302,7 +315,7 @@ def __str__(self):
         return self.name
 
     def to_dict(self):
-        data = {'name': self.name}
+        data: Dict[str, Any] = {'name': self.name}
```

Comment:
I don't think this helps. The method should get `-> dict`, though.

Reply:
I added the type hint to suppress some reported type errors from VScode, with the first assignment, it assumes that keys and values for the dict are always strings raising errors all over the place. 

Will remove though.